vm_supervisor/vm/firecracker/instance.py

```diff
@@ -50,14 +52,14 @@ class AlephFirecrackerInstance(AlephFirecrackerExecutable):
     is_instance = True
 
     def __init__(
-        self,
-        vm_id: int,
-        vm_hash: ItemHash,
-        resources: AlephInstanceResources,
-        enable_networking: bool = False,
-        enable_console: Optional[bool] = None,
-        hardware_resources: MachineResources = MachineResources(),
-        tap_interface: Optional[TapInterface] = None,
+            self,
+            vm_id: int,
+            vm_hash: ItemHash,
+            resources: AlephInstanceResources,
+            enable_networking: bool = False,
+            enable_console: Optional[bool] = None,
+            hardware_resources: MachineResources = MachineResources(),
+            tap_interface: Optional[TapInterface] = None,
```

Comment:
Please make sure that the PR passes the code quality tests: https://github.com/aleph-im/aleph-vm/actions/runs/5323885887/jobs/9642375702

`black`, `isort` and `mypy` all report errors on these changes.

I know that our global test suite is broken due to issues with the testing on virtual servers, but the rest of the tests should pass.